### ISSUE TITLE
Looking for maintainers - please reach out!

### ISSUE THREAD

**Issue by raix** [raix/push#356]

**Comment by da314pc** [comment]
@raix I am interested. I use this package heavily.

**Comment by shivang007** [comment]
Sounds interesting and a good idea, as raix/push is highly used in meteor community it needs maintenance.

**Comment by raix** [comment]
@da314pc and @shivang007 do both of you want to be added as maintainers? (I'll need your Meteor developer login name for allowing you to publish)

**Comment by shivang007** [comment]
That will be awesome @raix 
My Meteor dev username: _shivangkar7_

**Comment by da314pc** [comment]
@raix  My meteor dev username: da314pc

**Comment by raix** [comment]
@shivang007 and @da314pc  I have added you to the repository and meteor group - so you should be able to release new versions etc. let me know if it doesn't work or if you need more information on the package. Thanks guys

**Comment by aboire** [comment]
@da314pc @shivang007 do you have to watch this pull request https://github.com/raix/push/pull/354 which allows to use fcm for android and ios

**Comment by da314pc** [comment]
@aboire that has definitely been something I wanted to add to this library. 

As far as the next release there a couple things I think we have to tackle.

1) Better integration for android 8.0 oreo
2) take a look at the APN library, there has been so many changes and reported issues we need to upgrade to a stable version
https://github.com/node-apn/node-apn/issues/557#issuecomment-340812136
3) FCM for ios could be more reliable on ios, but we should still keep APN support.

**Comment by aboire** [comment]
@da314pc for the 2: apn it might be necessary to add this pull request https://github.com/raix/push/pull/327

**Comment by shivang007** [comment]
> @da314pc @shivang007 do you have to watch this pull request #354 which allows to use fcm for android and ios

@aboire Yes that is definitely on the to-do list

**Comment by raix** [comment]
@aboire do you want to be part of the maintainer group?

**Comment by aboire** [comment]
@raix if i can help yes

**Comment by raix** [comment]
@aboire I've added you to the repo - lmk if you need publish access

**Comment by shivang007** [comment]
@raix Can we please have a slack/spectrum/any group to improve our communication gap?

**Comment by raix** [comment]
@shivang007 It makes sense - are you guys setting it up or should I?

**Comment by shivang007** [comment]
@raix You set it up, we will join.

**Comment by raix** [comment]
I've created https://meteoropensourcecomm.slack.com - and added a channel for "meteor-push" @da314pc I don't have your email to send an invite - @shivang007 @aboire Please help invite

**Comment by da314pc** [comment]
@raix I have sent you an email.

**Comment by BrianMulhall** [comment]
is there still a need for help? i would like to contribute if possible

**Comment by da314pc** [comment]
@BrianMulhall yea, more people are always better. We need better docs for the new apn tokens, and also alot fo people have migrated to react natvie, but this project is still great for a back-end service. So i wanted to have a more update to guide for them,

**Comment by BrianMulhall** [comment]
if i were to fork this repo and then familiarize myself with it would that include all the code/doc/tests/examples/etc that support the project? or are there other areas that contain relevant information?

**Comment by BrianMulhall** [comment]
and the react-native users no longer use meteor for their applications but could still make use of this code running (server side) on node for their new projects? I'm not sure I am following that comment 100 percent

**Comment by da314pc** [comment]
Yes you would be familiar with this project.  As far as react native, This project has 2 parts, 
the client - which manages all of the token registration, token removal, 
the server- which integrates node fcm, & apple apn services into a nice clean api for meteorjs

we can still use the server, but i think there has been some confusion on how to integrate it.

**Comment by BrianMulhall** [comment]
ok cool, let me get up to speed with the repo and take a look around and I am going to try and get it working with a meteor project i have set up just to see it action. then ill reach out and see how i can be of any help.

**Comment by da314pc** [comment]
Awesome, the hardest part is the setup. Apple's push service wont really work until you have a beta app out, through the app store, but google's fcm service will work just running the app on an android phone. Just remember to have the Server settings set to dev.

**Comment by BrianMulhall** [comment]
i am able to get it working through firebase cloud messaging on an android device running 8.1 with phonegap-plugin-push version 2.2.3 as my cordova plugin. Worked liked a charm. so next step for me is to take a look around the code and then do a bit of a ramp up on how push actually works (like the internals of it all). just so i know whats what

**Comment by BrianMulhall** [comment]
i think if i could help out with documentation and maybe testing until i am familiar with things that would be a good place to start. I cant do much damage haha and ill learn the project

**Comment by StorytellerCZ** [comment]
If anyone is willing to take on maintaining this project please ping me!
